lib/order.rb

```diff
@@ -0,0 +1,44 @@
+require_relative 'text'
+
+class Order
+
+  attr_accessor :menu, :text
+
+  def initialize(text = Text_message.new)
+    @menu = @menu = {
+      "Sushi set" => 12,
+      "Prawn katsu curry" => 9,
+      "Kitsune udon" => 7,
+      "Onigiri" => 3,
+      "Matcha ice-cream" => 4
+    }
+    @total = 0
```

Comment:
Does this need to be an instance variable?

Instance variable should only be used when you need to store the state of an object as a whole, and that state has the same shelf life as the object.
If you just need to store information for a short amount of time, use a local variable. If you need to pass information between a method and another method called inside it, use an argument. Using an instance variable that is meant to be “short lived”, just to be used in a single method and none after is a source of bugs.